js/src/personal/kazi/components/Hero/Hero.module.css

```diff
@@ -0,0 +1,47 @@
+.wrapper {
```

Comment:
Folder name should be lowercase.

Reply:
Changed all component folder names to being lowercase